src/software/ai/navigator/path_planner/hrvo/lv_agent.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "software/ai/navigator/path_planner/hrvo/agent.h"
+#include "software/ai/navigator/path_planner/hrvo/hrvo_agent.h"
+#include "software/ai/navigator/path_planner/hrvo/lv_agent.h"
+#include "software/ai/navigator/path_planner/hrvo/robot_path.h"
+#include "software/ai/navigator/path_planner/hrvo/velocity_obstacle.h"
+#include "software/geom/vector.h"
+#include "software/world/robot_state.h"
+#include "software/world/team_types.h"
+
+class LVAgent : public Agent
```

Comment:
Could you add a comment about what LVAgent is. Not sure if we really explain that it's LinearVelocity and what that means

Reply:
added to constructor javadoc